Cargo.toml.jinja

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "frequenz-{{project_name}}"
+description = "{{project_description}}"
+version = "0.1.0"
+edition = "2021"
+authors = ["Frequenz Energy-as-a-Service GmbH"]
+{% if open_source %}
+license = "MIT"
+{% else %}
+license = "proprietary"
+{% endif %}
+
+{% if project_type == "lib" %}
+[lib]
+name = "frequenz_{{project_name | replace("-", "_")}}"
+path = "src/lib.rs"
```

Comment:
For `bin` projects, is the location of the binary already known by convention in rust?

Reply:
it is `src/main.rs` by convention. `src/lib.rs` is actually also already known in advance. Let me add a section for `bin` anyways.